rc/filetype/ocaml.kak

```diff
@@ -74,4 +74,22 @@ evaluate-commands %sh{
   "
 }
 
+# Conveniences
+# ‾‾‾‾‾‾‾‾‾‾‾‾
+
+# C has header and source files and you need to often switch between them.
+# Similarly OCaml has .ml (implementation) and .mli (interface files) and
+# one often needs to switch between them.
+#
+# This command provides a simple functionality that allows you to accomplish this.
+define-command ocaml-ml-switch-mli -docstring 'Switch between .ml and .mli file or vice versa' %{
```

Comment:
This should probably be called `ocaml-alternative-file`, and set the `alt` alias at window scope.
See `go-alternative-file` for an example, their error messages are also nice.

Reply:
Done! 

I don't think any error messages are needed for OCaml though...